plugin/hotswap-agent-spring-boot-plugin/src/main/java/org/hotswap/agent/plugin/spring/boot/env/v2/ListPropertySourceReloader.java

```diff
@@ -0,0 +1,67 @@
+package org.hotswap.agent.plugin.spring.boot.env.v2;
+
+import org.hotswap.agent.logging.AgentLogger;
+import org.hotswap.agent.plugin.spring.boot.env.HotswapSpringPropertiesReloader;
+import org.hotswap.agent.plugin.spring.boot.env.HotswapSpringReloadMap;
+import org.springframework.core.io.Resource;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * it is not forbidden to extend this class, because there are some class conflict.
+ *
+ * @param <T>
+ */
+public class ListPropertySourceReloader<T> implements HotswapSpringPropertiesReloader<List<Map<String, T>>> {
```

Comment:
'ListPropertySourceReloader', 'HotswapSpringProperties', 'HotswapSpringReloadMap' should be group together into the same sub package. The reason is they all implement HotswapSpringPropertiesReloader.